clients/src/main/java/org/apache/kafka/common/PartitionInfo.java

```diff
@@ -80,13 +80,10 @@ public String toString() {
     /* Extract the node ids from each item in the array and format for display */
     private String fmtNodeIds(Node[] nodes) {
```

Comment:
Can you please rename this to `formatNodeIds`. The name is unnecessarily cryptic (unrelated to this PR, I know).